site/config-overrides.md

```diff
@@ -205,61 +205,61 @@ The following `jdbc-testDS.xml` override template demonstrates setting the URL,
 ```
 
 ---
-# Step-by-Step Guide
+# Step-by-step guide
 
-* Create a directory containing (A) a set of situational configuration templates for overriding the mbean properties you want to replace and (B) a version.txt file.
+* Create a directory containing (A) a set of situational configuration templates for overriding the MBean properties you want to replace and (B) a `version.txt` file.
   * This directory must not contain any other files.
-  * The version.txt file must contain only the string `2.0`.
-  * Templates must not override the settings listed in [Unsupported Overrides](#unsupported-overrides).
-  * Templates must be formatted and named as per [Override Template Names and Syntax](#override-template-names-and-syntax) and [References](#references).
-  * Templates can embed macros that reference environement variables or Kubernetes secrets.  See [Override Template Macros](#override-template-macros).
-* Create a kubernetes config map from the directory of templates.
-  * The config map must be in the same kubernetes namespace as the domain.
-  * If the config map is going to be used by a single DOMAIN_UID, it is recommended to add the 'weblogic.domainUID=<mydomainuid>' label to help track the resource.
-  * For example, assuming './mydir' contains your version.txt and situation config template files:
+  * The `version.txt` file must contain only the string `2.0`.
+  * Templates must not override the settings listed in [Unsupported overrides](#unsupported-overrides).
+  * Templates must be formatted and named as per [Override template names and syntax](#override-template-names-and-syntax) and [References](#references).
+  * Templates can embed macros that reference environement variables or Kubernetes secrets.  See [Override template macros](#override-template-macros).
+* Create a Kubernetes configuration map from the directory of templates.
+  * The configuration map must be in the same Kubernetes namespace as the domain.
+  * If the configuration map is going to be used by a single `DOMAIN_UID`, it is recommended to add the `weblogic.domainUID=<mydomainuid>` label to help track the resource.
+  * For example, assuming `./mydir` contains your `version.txt` and situation configuration template files:
     ```
     kubectl -n MYNAMESPACE create cm MYCMNAME --from-file ./mydir
     kubectl -n MYNAMESPACE label cm MYCMNAME weblogic.domainUID=DOMAIN_UID
     ```
-* Create any kubernetes secrets referenced by a template macro.
-  * Secrets can have multiple keys (files) that can hold either cleartext or base64 values
-  * Secrets must be in the same kubernetes namespace as the domain
-  * If a secret is going to be used by a single DOMAIN_UID, it is recommended to add the 'weblogic.domainUID=<mydomainuid>' label to help track the resource.
+* Create any Kubernetes secrets referenced by a template macro.
+  * Secrets can have multiple keys (files) that can hold either cleartext or base64 values.
```

Comment:
Secrets can have multiple keys (files) that can hold either cleartext or base64 values. 

must be reworded to enforce password should be in base64 encoding.

Reply:
Rosemary, you can add something like '(It's recommended to use base64 values for passwords via 'Opaque' type secrets in their 'data' field, just so that they can't be easily read at a casual glance.  See https://kubernetes.io/docs/concepts/configuration/secret/ for more information.)'.

Reply:
Got it.